lib/pure/strutils.nim

```diff
@@ -241,7 +235,7 @@ proc toUpperAscii*(c: char): char {.noSideEffect,
   ## character.
   ##
   ## See also:
-  ## * `isLowerAscii proc<#isLowerAscii,char>`_
+  ## * `isUpperAscii func<#isUpperAscii,char>`_
```

Comment:
original link is wrong

Reply:
and that's why we need https://github.com/nim-lang/RFCs/issues/125

Reply:
No really 
I mean it should use `isUpperAscii` instead of `isLowerAscii`.

Reply:
I know, I'm referring to fact that a link like `isLowerAscii proc<#isLowerAscii,char>`_ (which could be in a different file or even different project) would break after this PR, because of s/proc/func. As far as this PR goes, that's an acceptable tradeoff, but with https://github.com/nim-lang/RFCs/issues/125 this would not be a problem.